app/models/agents/java_script_agent.rb

```diff
@@ -114,12 +114,9 @@ def execute_js(js_function, incoming_events = [])
       context["getOptions"] = lambda { |a, x| interpolated.to_json }
       context["doLog"] = lambda { |a, x| log x }
       context["doError"] = lambda { |a, x| error x }
-      context["getMemory"] = lambda do |a, x, y|
```

Comment:
You are right, that's not really the scope of this PR.

I was wondering how that even works, so V8 has to check the arity of the lambda and fill the missing arguments with nil?